pkgs/tools/misc/svtplay-dl/default.nix

```diff
@@ -4,21 +4,22 @@
 let
 
   inherit (python3Packages)
-    python pytest nose cryptography pyyaml requests mock python-dateutil setuptools;
+    python pytest nose cryptography pyyaml requests mock requests-mock
+    python-dateutil setuptools;
 
 in stdenv.mkDerivation rec {
   pname = "svtplay-dl";
-  version = "4.2";
+  version = "4.3";
 
   src = fetchFromGitHub {
     owner = "spaam";
     repo = "svtplay-dl";
     rev = version;
-    sha256 = "1bsinf2r07g8c03mcw4gprl92wmysyaa81s8wyck0c6wdq3hcsnp";
+    sha256 = "04gmiqg6mq96fw4v1v6947i216fsf0vcnjp3ya8bbii5na91y1aa";
   };
 
   pythonPaths = [ cryptography pyyaml requests ];
-  buildInputs = [ python perl mock python-dateutil setuptools ] ++ pythonPaths;
+  buildInputs = [ python perl mock requests-mock python-dateutil setuptools ] ++ pythonPaths;
```

Comment:
You are sure this is not a checkInput?

Reply:
`checkInputs` also work, doesn't make much difference in this case. It'll end up a run-time dependency regardless. In any case, moved mock and requests-mock to `checkInputs` since it definitely makes more semantic sense.